packages/atoms/src/Radio.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from 'react';
+import {
+    NativeStyleObject,
+    NativeStyleUtils,
+    prepareNativeStyle,
+    useNativeStyles,
+} from '@trezor/styles';
+import { Pressable, View } from 'react-native';
+
+type RadioValue = string | number | undefined;
+
+interface RadioProps {
+    value: string;
+    isChecked?: boolean;
+    onPress?: (value: RadioValue) => void;
+    isDisabled?: boolean;
+    style?: NativeStyleObject;
+}
+
+type RadioStyleProps = {
+    isChecked?: boolean;
+    isDisabled?: boolean;
+};
+
+const getRadioBorderColor = (
+    utils: NativeStyleUtils,
+    { isChecked, isDisabled }: RadioStyleProps,
+) => {
+    if (isChecked && !isDisabled) {
+        return utils.colors.green;
+    }
+    return utils.colors.gray400;
+};
+
+const radioStyle = prepareNativeStyle<RadioStyleProps>((utils, { isChecked, isDisabled }) => ({
+    height: 24,
+    width: 24,
+    backgroundColor: isDisabled ? utils.colors.gray400 : utils.colors.white,
+    flexDirection: 'row',
+    alignItems: 'center',
+    justifyContent: 'center',
+    borderRadius: utils.borders.radii.round,
+    borderWidth: isChecked ? utils.borders.widths.lg : utils.borders.widths.md,
+    borderColor: getRadioBorderColor(utils, { isChecked, isDisabled }),
+}));
+
+const radioCheckStyle = prepareNativeStyle<RadioStyleProps>((utils, { isDisabled }) => ({
+    height: 14,
+    width: 14,
+    borderRadius: utils.borders.radii.round,
+    backgroundColor: isDisabled ? utils.colors.gray600 : utils.colors.forest,
+}));
+
+export const Radio = ({
+    value,
+    isChecked: isCheckedProp,
+    onPress,
+    isDisabled,
+    style,
+}: RadioProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    const [isChecked, setIsChecked] = useState<boolean>(isCheckedProp ?? false);
+
+    const triggerRadioChange = () => {
+        if (!isDisabled) {
+            setIsChecked(innerIsChecked => !innerIsChecked);
+            onPress?.(value);
+        }
+    };
+
+    return (
+        <Pressable android_ripple={{}} disabled={isDisabled} onPress={triggerRadioChange}>
```

Comment:
please do not use Pressable anywhere by default, use rather TouchableOpacity/Highlight. Use of Pressable will make sense when we will going to create our animations, but before that is better to have at least some animation that no animation at all.

Reply:
Ok, no problem. But I am just curious why is then Pressable used in our Button component?

Reply:
I thinked that we will implement animations very soon, but I there will be some custom designed animations and it will took some time to design them. So you can fix this also in Button if you want.